recipe/meta.yaml

```
@@ -1,12 +1,12 @@
-{% set version = "3.9.0" %}
+{% set version = "3.12.0" %}
 
 package:
   name: flit-suite
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/f/flit/flit-{{ version }}.tar.gz
```

```suggestion
  url: https://pypi.org/packages/source/f/flit/flit-{{ version }}.tar.gz
```
The preferred TLD for pypi is `.org`